test/Microsoft.AspNet.Mvc.TagHelpers.Test/CacheTagHelperTest.cs

```diff
@@ -320,6 +318,47 @@ public void GenerateKey_WithMultipleVaryByOptions_CreatesCombinedKey()
         }
 
         [Fact]
+        public async Task ProcessAsync_DoesNotUseCache_OnPassingFalseWhenEnabled()
+        {
+            // Arrange
+            var cache = new Mock<IMemoryCache>();
+            cache.Setup(c => c.Set(
+                /*key*/ It.IsAny<string>(),
+                /*link*/ It.IsAny<IEntryLink>(),
+                /*state*/ It.IsAny<object>(),
+                /*create*/ It.IsAny<Func<ICacheSetContext, object>>()))
+                .Returns(new DefaultTagHelperContent().SetContent("ok"))
+                .Verifiable();
+            object cacheResult;
+            cache.Setup(c => c.TryGetValue(It.IsAny<string>(), It.IsAny<IEntryLink>(), out cacheResult))
+                .Returns(false);
```

Comment:
does test fail without this setup?  expected cache to be completely unused when tag helper is disabled.